api/src/main/java/org/openmrs/module/bedmanagement/service/impl/BedManagementServiceImpl.java

```diff
@@ -321,6 +358,16 @@ public void deleteBedType(BedType bedType) {
 		bedManagementDao.deleteBedType(bedType);
 	}
 	
+	@Override
+	public void deleteBedPatientAssignment(BedPatientAssignment bpa, String reason) {
```

Comment:
Is there reason we don't just call this "voidBedPatientAssignment" (is there an issue with the Void aop).  Usually in OpenMRS (I think) delete methods do actual hard deletes.

Reply:
it probably won't, but worth testing?  And if it doesn't I'm fine with sticking with "delete" instead of "void" for internal consistency in the module.

Reply:
yeah... that didn't work. I'll keep it as is.